### PR TITLE
Add NeuropixelsV2e GUI documentation

### LINKED ISSUE #41: Create documentation page for NeuropixelsV2e GUI
Currently there is no documentation for any dialogs created in the Design library.

First, we should create a page for the `NeuropixelsV2e` dialogs based on the format discussed in #37 to test how the format looks.

One aspect not discussed there, for all of the GUIs created so far there are nested layers to them, meaning that for `NeuropixelsV2e` specifically there is a dialog window for an individual probe configuration, which can be combined with a second probe configuration at the device level, which can then be combined with a `NeuropixelsV2eBno055` dialog at the headstage level. Each level should be discussed, specifically how to access them (i.e., double-clicking the headstage node, selecting the ellipses next to the probe configuration variable, etc.). While the bulk of the work is done at the lowest level (probe configuration dialog), the major screenshot to discuss should be the headstage level to be the most comprehensive.

This page should include:
- Screenshot of the GUI
  - This includes all levels
  - Screenshot should be saved in an easily editable form (see [README](https://github.com/open-ephys/onix1-bonsai-docs?tab=readme-ov-file#creating-edited-screenshots) for details) with sections highlighted for reference
- Text describing each of the sections highlighted in the screenshot
- Minimal GIFs, but if text is cumbersome to use or the action has a temporal component that benefits from a video format then use that
- Links to relevant information already provided (e.g., enums for dropdown menus should be referenced, not written manually)
- Minimal workflow that contains the nodes which have GUIs attached to them
  - Call the same workflow as the aggregate page

### LINKED ISSUE #49: Add a way in the template to center images?
Is there any way to modify the template or to add some CSS that will center images? Using the default format of `Title` attempts to stretch the image from end to end, which makes it appear centered, but if we want to set a particular width (i.e. ,`Title{width=600px}`) then it will always be a left-aligned image. 

It is possible cheat a little bit and directly inject HTML into the markdown file, but this goes against the [philosophy of Markdown](https://stackoverflow.com/questions/12090472/how-do-i-center-an-image-in-the-readme-md-file-on-github/12091563#12091563). Adding HTML like this works

```
<p align="center">
  <img src="../link/to/image.png">
</p>
```

But this doesn't look very clean in the Markdown file, and also might not be properly rendered depending on what is serving the page.

My preferred method would be to add an `align="center"` parameter after the pixel width if possible (i.e., `Title{width=600px,align="center"}`), I think this keeps it as clean as possible while allowing for some customizability. However, I genuinely do not know if this is possible on the backend, hence the Question label.

### PR DESCRIPTION
**Pull request by bparks13**
Adding a tutorial for how to use the `NeuropixelsV2e` GUI.

Closes #41 
Closes #49

### CODE REVIEW

File: tutorials/neuropixelsv2e-gui-tut.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+When the GUI is first opened, and after a probe calibration file has been specified, the default [ProbeInterface](#probeinterface) configuration is loaded and drawn in the main panel. In this case, the default configuration is for a quad-shank `NeuropixelsV2e` probe, with the `Shank0BankA` channel preset selected. To load a new configuration, or load the default configuration, or save the current configuration, go to the File drop-down menu (see below) and choose the relevant option.
+
+<p align="center">
+  <img src="../images/neuropixelsv2e-gui-tut/probeconfiguration-filemenu.png">
+</p>
+
+#### Save ProbeInterface file
+
+To save a [ProbeInterface](#probeinterface) JSON file fully describing the probe, including which electrodes are currently enabled, go to the File drop-down menu, and select `Save Channel Configuration`. This will open a file dialog window to save the new JSON file. Choose a folder location and a name for the file, then hit `Save`. This will export the current channel configuration, and is a useful way to save any manually enabled electrodes as a backup, or to easily switch between different configurations between recordings.
```
Comment (cjsha) [inline tutorials/neuropixelsv2e-gui-tut.md:149]: Nitpick. I would split the last sentence here into two at the first comma, but it's clear to me either way.

File: tutorials/neuropixelsv2e-gui-tut.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+For `NeuropixelsV2eHeadstage`, the GUI allows for an easy way to change settings and visualize the effect. From the GUI, you can:
+
+- Modify `Probe A` and `Probe B` independently
+    - Choose the probe calibration file
+    - Change the [reference](xref:OpenEphys.Onix1.NeuropixelsV2QuadShankReference) for all electrodes
+    - Choose pre-defined channel presets or manually define within a constrained set of possible combinations
+    - Easily visualize which electrodes are enabled
+- Enable and disable the <xref:OpenEphys.Onix1.NeuropixelsV1eBno055Data> stream
+
+There are two ways to access configuration dialogs; 1) at the individual `NeuropixelsV2eProbeConfiguration` level where either `Probe A` or `Probe B` can be modified by itself, and 2) at the headstage level where both probes can be modified, as well as the `NeuropixelsV2eBno055` device.
```
[the remark this answers is not in the document]
Comment (cjsha) [reply tutorials/neuropixelsv2e-gui-tut.md:16]: After reading through this doc, I understand better why you did it like that. I dunno. I still think we should "suppress" the device configure operators and their usage

**Review by cjsha — COMMENTED** [review]
I think this great. My comments are mostly nitpicks

File: tutorials/neuropixelsv2e-gui-tut.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bparks13) [reply tutorials/neuropixelsv2e-gui-tut.md:16]: None of this is referring to the device configure operator, only the underlying probe configuration variable, but maybe there is a better way to say it. In the docs, the image I use only has the headstage configure operator and shows expanding the property pane to access the probe configuration level. Is that confusing though? I thought it made a clear distinction between the two levels without even saying that there is a device configure operator that can be used. 

I will leave this conversation unresolved for now until we make a decision

File: tutorials/neuropixelsv2e-gui-tut.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+Whether the GUI is opened at the probe configuration level, or the headstage level, the usage will be almost identical. There are some additional tabs present at the headstage level, but the `Probe A` and `Probe B` tabs will act exactly the same as opening a probe configuration GUI for that specific probe.
+
+### Channel constraints
+
+For `NeuropixelsV2e`, there will always be 384 channels enabled across the entire probe. Therefore, when enabling electrodes (either [manually](#manually-enabling-electrodes) or using [channel presets](#channel-presets)), it is possible that previously enabled electrodes will be disabled. Additionally, if more than 384 electrodes are selected to be manually enabled only the last 384 channels will end up being enabled. It is therefore recommended to always double-check that the correct electrodes are enabled.
```
[the remark this answers is not in the document]
Comment (bparks13) [reply tutorials/neuropixelsv2e-gui-tut.md:22]: Re: channel vs. electrode. These are two separate things; there are over 5,000 electrodes in the quad-shank probe (physical objects on the probe), meanwhile each electrode can be enabled to record at a particular channel (abstract concept).

File: tutorials/neuropixelsv2e-gui-tut.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cjsha) [reply tutorials/neuropixelsv2e-gui-tut.md:16]: Nevermind, I get it now. I think it's fine actually.

**Comment by cjsha** [comment]
I will say that it feels long to me, but I think it's good to get all this material out. If others agree that it feels long, we can trim it later if necessary/possible.

**Review by cjsha — APPROVED** [review]
Looks good to me.

I will say that it does feel long, but I think it's good to get all this material out. If others agree that it feels long, we can try trimming it later when reorganizing content per issue #40

**Comment by bparks13** [comment]
Agreed that it's a bit long, might be worth chopping it up and putting into specific pages, as per our discussions. We can revisit this when we start reorganizing the structure of the docs.

**Pull request merged**